### PR TITLE
Kotlin  CleanUp

### PR DESCRIPTION
**Pull request by jals413**
Kotlin Cleanup: integerListToArray #10530

## Fixes
Replaced the method integerListToArray to Kotlin method .toIntArray( ) , also removed the method integerListToArray 

Changes were made to DeckPickerContextMenu.kt and CustomStudyDialog.kt  files

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.kt [hunk 1]
```diff
@@ -116,7 +116,7 @@ class DeckPickerContextMenu(private val collection: Collection) : AnalyticsDialo
             }
             itemIds.add(CONTEXT_MENU_CREATE_SHORTCUT)
             itemIds.add(CONTEXT_MENU_DELETE_DECK)
-            return ContextMenuHelper.integerListToArray(itemIds)
+            return ContextMenuHelper.itemIds.toIntArray()
```
Comment (ShridharGoel) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.kt:119]: This needs to be fixed.

**Review by ShridharGoel — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.kt (hunk 1 above)
Comment (jals413) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.kt:119]: okay , trying

**Comment by jals413** [comment]
I did run it on my device , it was working fine

**Comment by jals413** [comment]
ok I will fix it , need some time

**Comment by mikehardy** [comment]
superceded by #10664

**Pull request closed without merging**